### PR TITLE
modeler move() method

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: pyaedt/modeler/Modeler.py [hunk 1]
```diff
@@ -1784,6 +1784,34 @@ def mirror(self, objid, position, vector):
...
 
+    @aedt_exception_handler
+    def move(self, objid, vector):
+        """Move a selection.
+
+        Parameters
+        ----------
+        objid : str, int, or Object3d
+            Name or ID of the object.
+        vector : float
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1795]:
```suggestion
        vector : list
```

File: pyaedt/modeler/Modeler.py [hunk 2]
```diff
@@ -1784,6 +1784,34 @@ def mirror(self, objid, position, vector):
...
+            List of the ``[x1, y1, z1]`` coordinates or
+            the Application.Position object for the vector. For 2D Extractor designs, 'z1' should be set to "0".
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        selections = self.convert_to_selections(objid)
+        Xvec, Yvec, Zvec = self.primitives._pos_with_arg(vector)
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1805]:
```suggestion
        x_vec, y_vec, z_vec = self.primitives._pos_with_arg(vector)
```

File: pyaedt/modeler/Modeler.py [hunk 3]
```diff
@@ -1784,6 +1784,34 @@ def mirror(self, objid, position, vector):
...
+
+        vArg1 = ["NAME:Selections", "Selections:=", selections, "NewPartsModelFlag:=", "Model"]
+        vArg2 = ["NAME:TranslateParameters"]
+        vArg2.append("TranslateVectorX:="), vArg2.append(Xvec)
+        vArg2.append("TranslateVectorY:="), vArg2.append(Yvec)
+        vArg2.append("TranslateVectorZ:="), vArg2.append(Zvec)
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1811]:
```suggestion
        vArg2.append("TranslateVectorX:="), vArg2.append(x_vec)
        vArg2.append("TranslateVectorY:="), vArg2.append(y_vec)
        vArg2.append("TranslateVectorZ:="), vArg2.append(z_vec)
```

File: pyaedt/modeler/Modeler.py [hunk 4]
```diff
@@ -1750,7 +1750,7 @@ def mirror(self, objid, position, vector):
         ----------
         objid : str, int, or Object3d
             Name or ID of the object.
-        position : float
+        position : int or float
             List of the ``[x, y, z]`` coordinates or the
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1754]:
```suggestion
        position : list
            List of int or float of the ``[x, y, z]`` coordinates or the
```

File: pyaedt/modeler/Modeler.py [hunk 5]
```diff
@@ -1784,6 +1784,34 @@ def mirror(self, objid, position, vector):
 
         return True
 
+    @aedt_exception_handler
+    def move(self, objid, vector):
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1788]:
```suggestion
    def move(self, object_id, vector):
```

File: pyaedt/modeler/Modeler.py [hunk 6]
```diff
@@ -1784,6 +1784,34 @@ def mirror(self, objid, position, vector):
...
+        """Move a selection.
+
+        Parameters
+        ----------
+        objid : str, int, or Object3d
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1793]:
```suggestion
        object_id : str, int, or Object3d
```

File: pyaedt/modeler/Modeler.py [hunk 7]
```diff
@@ -1784,6 +1784,34 @@ def mirror(self, objid, position, vector):
...
+        vector : float
+            List of the ``[x1, y1, z1]`` coordinates or
+            the Application.Position object for the vector. For 2D Extractor designs, 'z1' should be set to "0".
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        selections = self.convert_to_selections(objid)
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1804]:
```suggestion
        selections = self.convert_to_selections(object_id)
```

**Review by MaxJPRey — APPROVED** [review]

**Comment by MaxJPRey** [comment]
@ring630 I just proposed some minor modifications about the variable's names to try to follow the 'snake_case' convention. 
LGTM.
Thanks for adding the unit test too.

**Comment by ring630** [comment]
@MaxJPRey Thanks for the view. The method move was already there but it is called translate. I will copy the content from translate and add a deprecation warning in translate method.

**Comment by MaxJPRey** [comment]
@ring630 Sounds good.

**Pull request merged**